src/material/card/card.scss

```diff
@@ -87,7 +87,7 @@ $mat-card-header-size: 40px !default;
   .mat-button,
   .mat-raised-button,
   .mat-stroked-button {
-    margin: 0 8px;
+    margin: 0 8px 0 0;
```

Comment:
This won't work correctly on RTL layouts. You can test it out in the dev app by clicking the "RTL" button in the top right.